rules/dead-code/tests/Rector/Assign/RemoveUnusedAssignVariableRector/Fixture/skip_assign_expression_class_use_variable.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DeadCode\Tests\Rector\Assign\RemoveUnusedAssignVariableRector\Fixture;
+
+final class SkipAssignExpressionClassUseVariable
+{
+    public function run(string $foo = null, string $bar = null)
```

Comment:
```suggestion
    public function run()
```

Test fixture case should be as minimal as possible to make it easier in 2 years to debug them.

From the code I assume this is never used to test as tested variable name is `datetime`

Reply:
You're right, missed during some testing.